src/style.css

```diff
@@ -1,3 +1,79 @@
+* {
```

Comment:
Using the universal selector `*` can lead to performance issues as it applies styles to all elements. Consider applying these styles more selectively to improve performance.